features/features.go

```diff
@@ -94,6 +96,8 @@ var featureNames = map[SnapdFeature]string{
 	CheckDiskSpaceInstall: "check-disk-space-install",
 	CheckDiskSpaceRefresh: "check-disk-space-refresh",
 	CheckDiskSpaceRemove:  "check-disk-space-remove",
+
+	RefreshControl: "refresh-control",
```

Comment:
as I said elsewhere we used this name already for something else (the original refresh control via validation assertion); so we need to think a name for this

Reply:
the easiest is maybe to call it based on the hook:  "gate-auto-refresh-hook" ?

Reply:
Yes, that makes sense, +1

Reply:
Done.